library/src/main/kotlin/com/chuckerteam/chucker/internal/support/RequestProcessor.kt

```diff
@@ -75,4 +82,15 @@ internal class RequestProcessor(
                 null
             }
         }.firstOrNull()
+
+    private fun inferGqlRequest(body: String): String? {
```

Comment:
Inferring GraphQL from a body won't work for GET requests as there is only URL query parameter for those.

It's a bit hard to have a full-proof solution as anyone could have query parameters or bodies that conform to GraphQL format. I have an idea of how to deal with it but let's just first implement handling of GET requests as well.